container/example/example.tf

```diff
@@ -72,10 +88,30 @@ module "target" {
     port     = "80"
   }]
 
-  tags {
-    terraform   = "True"
-    environment = "dev"
+  tags = "${var.tags}"
+}
+
+# Create a target group with listeners.
+module "targetTCP" {
+  source = "github.com/TeliaSoneraNorge/divx-terraform-modules//container/target"
+
+  prefix            = "${var.prefix}"
+  vpc_id            = "${module.vpc.vpc_id}"
+  load_balancer_arn = "${module.nlb.arn}"
+
+  target {
+    protocol        = "TCP"
+    port            = "9000"
+    health_protocol = "TCP"
+    health_port     = "traffic-port"
   }
+
+  listeners = [{
+    protocol = "TCP"
+    port     = "9001"
```

Comment:
Perhaps add multiple listeners here so we have that covered also? Especially if this is now the test ;)